opentelemetry-sdk/tests/test_configurator.py

```diff
@@ -665,9 +677,151 @@ def test_logging_init_exporter(self):
         getLogger(__name__).error("hello")
         self.assertTrue(provider.processor.exporter.export_called)
 
+    @patch.dict(environ, {}, clear=True)
```

Comment:
Missing a test passing an invalid log level value and getting INFO set.

Reply:
Added and switched default to NOTSET to match existing behavior.